docs/domain-model.md

```diff
@@ -0,0 +1,123 @@
+
+A connector is composed of 2 subsystems:
+
+| **Control Plane (First checking step)** | **Data Plane (Second checking step)** |
+|:----------------------------------------|:--------------------------------------|
+| User verification (access to data)      | Moves bits.                           |
+| Contract negotiation.                   | Big Data Tasks.                       |
+| Overseeing of Policy enforcement.       | Streaming.                            |
+| Provisioning management.                | Events.                               |
+
+<p style='text-align: justify;'> The connector processes requests asynchronously, which are useful for example in case one has to deal with lengthy data preparation</p>
+
+#### 2. Definitions related to the EDC Connector
+
+<p style='text-align: justify;'> When publishing and accessing to data in a dataspace, it is necessary to have at least 2 different components: One to publish or make data available, and another one which can access to that data. For this case, we will call them provider and consumer. </p>
+
+* **Data Provider**:
+
+  <p style='text-align: justify;'> The Data Provider makes data available for being exchanged between a Data Owner and a Data Consumer. The Data Provider
+  may log the details of the successful (or unsuccessful) completion of the transaction at a Clearing House (
+  intermediary that provides clearing and settlement services for all financial and data exchange transactions) to
+  facilitate billing or resolve a conflict. Furthermore, the Data Provider can use Data Apps to enrich or transform the
+  data in some way, or to improve its quality (Data Apps are specific applications that can be integrated into the data
+  exchange workflow between two or more participants in the Industrial Data Space)<sup>2</sup> The data owner can also be the data
+  provider, when it is not, then a contract is needed between the client and provider in order to start a transfer of
+  data. </p>
+
+* **Data consumer**:
+
+  <p style='text-align: justify;'>The Data Consumer receives data from a Data Provider. Before the connection to a Data Provider can be established, the
+  Data Consumer can search for existing datasets by making an inquiry at a Broker Service Provider (provides the required
+  metadata for the Data Consumer to connect to a Data Provider.). Alternatively (In case the information to connect with
+  the Data Provider is already known to the Data Consumer), the Data Consumer can establish a connection with a Data
+  Provider directly<sup>2</sup> The data user, can also be the data consumer.</p>
+
+![img_3.png](dataConsumerDiagram.png)
```

Comment:
I am not sure whether we should use an IDS diagram for explaining our domain model. Especially when this is about the integration of an EDC as IDS Connector in an ecosystem. The domain model document should explain our understanding of relations inside and outside the EDC. You could take a look at the [existing diagrams](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/tree/main/docs/diagrams).

Reply:
Hi @juliapampus, the code was already updated. Please check the newest version.